README.rst

```diff
@@ -59,6 +59,42 @@ Measuring code coverage, continuing the steps above:
     (venv) $ coverage html
     (venv) $ open htmlcov/index.html
 
+Local development
+-----------------
+
+The local development setup is based on Docker so you need the following installed in your system:
+
+* `docker`
+* `docker-compose`
+
+Also you need to edit your `/etc/hosts` file to resolve `testrp` and `testprovider` hostnames to `localhost`.
```

Comment:
You will also need to edit your hosts file to resolve `testrp` and `testprovider` hostnames to 127.0.0.1